<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_LoadCitra">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>380</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Citra</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_prosesCitra">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>380</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Proses Citra</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>321</width>
      <height>261</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>80</y>
      <width>331</width>
      <height>261</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuOperasi_Task">
    <property name="title">
     <string>Operasi Task</string>
    </property>
    <addaction name="actionBrightness"/>
    <addaction name="actionSimple_Contrast"/>
    <addaction name="actionContrast_Stretching"/>
    <addaction name="actionNegative_Image"/>
    <addaction name="actionBiner_Image"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOperasi_Task"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionBrightness">
   <property name="text">
    <string>Brightness</string>
   </property>
  </action>
  <action name="actionSimple_Contrast">
   <property name="text">
    <string>Contrast</string>
   </property>
  </action>
  <action name="actionContrast_Stretching">
   <property name="text">
    <string>Contrast_Stretching</string>
   </property>
  </action>
  <action name="actionNegative_Image">
   <property name="text">
    <string>Negative Image</string>
   </property>
  </action>
  <action name="actionBiner_Image">
   <property name="text">
    <string>Biner Image</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
